lisa/sut_orchestrator/azure/features.py

```diff
@@ -783,3 +785,20 @@ def get_azure_disk_type(disk_type: schema.DiskType) -> str:
     schema.DiskType.StandardHDDLRS: "Standard_LRS",
     schema.DiskType.StandardSSDLRS: "StandardSSD_LRS",
 }
+
+
+class Resize(AzureFeatureMixin, features.Resize):
```

Comment:
It needs some changes in Azure platform too. Search for SerialConsole or StartStop to see what's extra logic in Azure platform.

Reply:
Could you elaborate on this a bit please?